web/jobs/jobs.go

```diff
@@ -807,28 +848,29 @@ func purgeJobs(c echo.Context) error {
 	return c.JSON(http.StatusOK, map[string]int{"deleted": len(jobsToDelete)})
 }
 
-// Routes sets the routing for the jobs service
-func Routes(router *echo.Group) {
-	router.GET("/queue/:worker-type", getQueue)
-	router.POST("/queue/:worker-type", pushJob)
-	router.POST("/support", contactSupport)
-
-	router.POST("/triggers", newTrigger)
-	router.GET("/triggers", getAllTriggers)
-	router.GET("/triggers/:trigger-id", getTrigger)
-	router.GET("/triggers/:trigger-id/state", getTriggerState)
-	router.GET("/triggers/:trigger-id/jobs", getTriggerJobs)
-	router.PATCH("/triggers/:trigger-id", patchTrigger)
-	router.POST("/triggers/:trigger-id/launch", launchTrigger)
-	router.DELETE("/triggers/:trigger-id", deleteTrigger)
-
-	router.POST("/webhooks/bi", fireBIWebhook)
-	router.POST("/webhooks/:trigger-id", fireWebhook)
-
-	router.POST("/clean", cleanJobs)
-	router.DELETE("/purge", purgeJobs)
-	router.GET("/:job-id", getJob)
-	router.PATCH("/:job-id", patchJob)
+// Register all the `/jobs` routes to the given router
+func (h *HTTPHandler) Register(router *echo.Group) {
+	router.GET("/queue/:worker-type", h.getQueue)
+	router.POST("/queue/:worker-type", h.pushJob)
+	router.POST("/support", h.contactSupport)
+	router.POST("/campaign-emails", h.sendCampaignEmail)
```

Comment:
This endpoint allow to send ONE email per call, so I think it should be `/campaign-email` without S

Reply:
and the documentation has no final S for this endpoint

Reply:
The documentation is wrong, I'll fix that.

The naming follows the REST conventions (POST request to the resources route to create a resource) same as the `/jobs` route.